backend/test/modules/email-sending/core/application/SendEmail.mailhog-spec.ts

```diff
@@ -0,0 +1,38 @@
+import { MailhogClient } from 'mailhog-awesome';
+import { NodeMailerEmailSender } from '../../../../../src/modules/email-sending/infrastructure/mailer/NodeMailerEmailSender';
+
+describe('Example Jest email test-suite', () => {
```

Comment:
Tutaj jak opiszesz te testy lepiej to jest OK.
Przetestowałeś tutaj, że NodeMailerEmailSender rzeczywiście wysyła maila.

W SendEmailAfterProfileWasCreated.spec.ts testujesz (a raczej dalej próbujesz :)), ze po opublikowaniu eventu playerProfileWasCreated będzie command wysłany i to OK. 

No i potem zostanie ostatni test (już w module maila), że wysłanie commanda SendEmail powoduje wywołanie EmailSendera z odpowiednimi argumentami. Żeby zobaczyć czy wszystko się spina.

Reply:
> > @nowakprojects jest jeszcze jeden problem .... przestały się wysyłać maile jeżeli zrobię POST new player profile ... wcześniej działały i ogólnie maile się wysyłają przy starcie serwera kiedy jest tworzonych na sztywno kilka profilów
> > Nie mogę zlokalizować błędu :-( pomożesz?
> 
> A jest jakiś test co przechodził a nie przechodzi :)? Tak niestety ciężko szukać jak nie ma testów. Tak to pozostaje Ci debugowanie i zobacz jak Flow idzie inaczej. Co jest wywoływane itp.

kurde nic nie zmieniłem i dzisiaj działa -.- ... wtf?

Reply:
> Tutaj jak opiszesz te testy lepiej to jest OK.
> Przetestowałeś tutaj, że NodeMailerEmailSender rzeczywiście wysyła maila.
> 
> W SendEmailAfterProfileWasCreated.spec.ts testujesz (a raczej dalej próbujesz :)), ze po opublikowaniu eventu playerProfileWasCreated będzie command wysłany i to OK.
> 
> No i potem zostanie ostatni test (już w module maila), że wysłanie commanda SendEmail powoduje wywołanie EmailSendera z odpowiednimi argumentami. Żeby zobaczyć czy wszystko się spina.

"będzie command wysłany i to OK." - tego nie kumam... jak command wysłany? 
![image](https://user-images.githubusercontent.com/31566345/110605870-db9e0100-8189-11eb-8395-9495f3c2ae45.png)
?

Reply:
Gdzie to mockujesz :) ? Z tego co widzę testModuleCore używa InMemory. A w Twoim przypadku nie chcemy takiego działania.

Reply:
Za długo trochę się męczysz z tymi mailami a one nie sa must have. Zobacz może z chłopakami jeszcze i spórbujcie to skończyć szypciorem :) ? Bo nam logiki dużo brakuje, a na to możesz się z kimś sprawoać / dokończyć potem. I tak mamy neizależne moduły, wiec to nie spowoduje konfliktów jak poczekasz.

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/37/commits/17da06592298dd6c6d2ab51d48a89835800d2b38
no już chyba bliżej, ale nie wiem czemu niby tego nie łapie ...

Reply:
> Za długo trochę się męczysz z tymi mailami a one nie sa must have. Zobacz może z chłopakami jeszcze i spórbujcie to skończyć szypciorem :) ? Bo nam logiki dużo brakuje, a na to możesz się z kimś sprawoać / dokończyć potem. I tak mamy neizależne moduły, wiec to nie spowoduje konfliktów jak poczekasz.

no już został w zasadzie tylko ten test, gdzie kurde wystarczy dopisać jedną / dwie linijki ... a siedzę też ze wszystkimi :-)

Reply:
oj i na pewno tego mam serdecznie dosyć .... ale jak już tak niewiele brakuje to chcę to już skończyć